paparazzi/src/main/java/app/cash/paparazzi/PaparazziSdk.kt

```diff
@@ -347,7 +348,7 @@ public class PaparazziSdk @JvmOverloads constructor(
 
     // Execute the block at the requested time.
     System_Delegate.setNanosTime(0L)
-    System_Delegate.setNanosTime(frameNanos)
+    Choreographer_Delegate.sChoreographerTime = frameNanos
```

Comment:
Now that we are using `Choreographer` frame we need to make sure the time is correct.